website/blog/2018-06-25-on-consuming-and-publishing-es6+-packages.md

```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+title:  "On Consuming (and Publishing) ES6+ Packages"
+author: Henry Zhu
+authorURL: https://twitter.com/left_pad
+date:   2018-06-25 12:00:00
+categories: announcements
+share_text: "On Consuming (and Publishing) ES6+ Packages"
+---
+
+How can we make compiling dependencies not just possible, but normal?
```

Comment:
Yeah, good point. I should just move it to the point where it actually talks about the issue that got brought up regarding publishing proposal syntax, not need to reference since it works on it's own!